assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java

```diff
@@ -184,10 +179,14 @@ protected void configure() {
     //        bind(org.eclipse.che.api.agent.server.filters.AddExecInstallerInWorkspaceFilter.class);
     //        bind(org.eclipse.che.api.agent.server.filters.AddExecInstallerInStackFilter.class);
 
-    // FIXME: spi
-    install(new DockerInfraModule());
-    install(new LocalDockerModule());
-    install(new OpenShiftInfraModule());
+    String infrastructure = System.getenv("CHE_INFRASTRUCTURE_ACTIVE");
+    if ("openshift".equals(infrastructure)) {
```

Comment:
maybe we should think how to made that part more friendly for new types of infra... just in case if we will decide to port swarm.

Reply:
My intention was to make it possible to deploy SPI branch of Che on Minishift ASAP. 
AFAIK we don't see a use case with several infras impls bound and are going to have separate assemblies per infra, but don't want to introduce them now. So it is a temporary solution to start using Che on Minishift with minimal efforts.

Reply:
oki, thanks for info